webhooks/ratio_validator.go

```diff
@@ -91,6 +91,46 @@ func (v *RatioValidator) Handle(ctx context.Context, req admission.Request) admi
 	return admission.Allowed("ok")
 }
 
+func (v *RatioValidator) recodObject(ctx context.Context, r *Ratio, req admission.Request) (*Ratio, error) {
+	switch req.Kind.Kind {
+	case "Pod":
+		pod := corev1.Pod{}
+		if err := v.decoder.Decode(req, &pod); err != nil {
+			return r, err
+		}
+		r = r.RecordPod(pod)
+	case "Deployment":
+		deploy := appsv1.Deployment{}
+		if err := v.decoder.Decode(req, &deploy); err != nil {
+			return r, err
+		}
+		r = r.RecordDeployment(deploy)
+	case "StatefulSet":
+		sts := appsv1.StatefulSet{}
+		if err := v.decoder.Decode(req, &sts); err != nil {
+			return r, err
+		}
+		r = r.RecordStatefulSet(sts)
+	}
+	return r, nil
+}
+
+func (v *RatioValidator) isNamespaceDisabled(ctx context.Context, nsName string) (bool, error) {
+	ns := corev1.Namespace{}
+	err := v.client.Get(ctx, client.ObjectKey{
+		Name: nsName,
+	}, &ns)
+	if err != nil {
+		return false, err
+	}
+
+	disabled, ok := ns.Annotations[RatioValidatiorDisableAnnotation]
+	if !ok {
+		return false, err
```

Comment:
I prefer `nil` here so i don't have to back-reference what the variable means.

```suggestion
		return false, nil
```

Reply:
Good point. I agree